dms/views/dms_file.xml

```diff
@@ -71,8 +71,8 @@
                     <field
                         name="directory_id"
                         icon="fa-folder"
-                        context="{'directory_short_name': True}"
```

Comment:
Why removing this?

Reply:
I have separated in several commits the changes to understand it.